src/TraceEvent/TraceEvent.csproj

```diff
@@ -41,7 +40,7 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Debug\Microsoft.Diagnostics.Tracing.TraceEvent.xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>0649 0618</NoWarn>
+    <NoWarn>0649,0618</NoWarn>
```

Comment:
@vancem The #pragma is no more in this PR. Any objection to adding a `<NoWarn>` for the projects which still report that warning and filing an issue to address it later?